### PR TITLE
SPAG and expansion pass `organise/before`

### PR DESCRIPTION
**Pull request by AC-Aksan**
General SPAG and formatting pass of the marked section, includes some page expansion to Side Events and aims to improve judging with reference to HHEU/wiki#104.  Bundled due to the minor changes to most pages.

### CODE REVIEW

File: docs/organise/before/finance.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ## Budget
 
-Budgets are what makes the hackathon work properly. They provide refreshments, meals, prizes, etc... from Sponsors. Your budget will come from your[Sponsorship Tiers](/organise/sponsorship/#sponsorship-tiers) and at times, depending on budget history, any surplus from the previous hackathon's budget.
+Budgets are what makes the hackathon work properly. They provide refreshments, meals, prizes, etc. from sponsors. Your budget will come from your [Sponsors](/organise/sponsorship) and at times, depending on budget history, any surplus from the previous hackathon's budget.
```
Comment (robzwolf) [inline docs/organise/before/finance.md]: Can `[Sponsors]` have a lowercase `S`?

File: docs/organise/before/finance.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 
 Things to consider:
 
-* Not everyone is present at every meal.
-* Most hackathons have an attrition rate of 30-50%, usually around 40%.
+* Not everyone is present at every meal. (But Sponsors and Volunteers need food too.)
```
Comment (robzwolf) [inline docs/organise/before/finance.md]: Can we change `Sponsors` and `Volunteers` to `sponsors` and `volunteers`?

File: docs/organise/before/gettingstarted.md [hunk 3]
```diff
@@ -1,39 +1,39 @@
...
 
-Most hackathons have at least 3 different tiers which is a common trend between different hackathons.
+Most hackathons have at least 3 different tiers, often bronze, silver and gold, which is a common trend between different hackathons.
 
 
 ### [Major League Hacking (MLH)](https://mlh.io)
-If your creating a brand new Hackathon, MLH may be able to provide some key information and and financial support to ensure that the hackathon runs smoothly.
-Hackathons registered with MLH also provide additional support with events before, during & after the day. Check here for the process to make it an [MLH hackathon event](https://mlh.io/).
+If you're creating a brand new Hackathon, MLH may be able to provide some key information and financial support to ensure that the hackathon runs smoothly.
+Hackathons registered with MLH also recieve additional support with events before, during & after the day. Check here for the process to make it an [MLH hackathon event](https://mlh.io/).
```
Comment (robzwolf) [inline docs/organise/before/gettingstarted.md]: `recieve` should be `receive`

File: docs/organise/before/judging.md [hunk 4]
```diff
@@ -1,16 +1,30 @@
...
+- Organisers
+- University Faculty
+
+When selecting judges you should aim to have enough judges to cover all of your hacks in a reasonable timeframe. Bias should be avoided if possible.  
+
+## How to structure judging?
+Allocate adequate time given your number of judges and attendees to allow the judges adequate time to consider each hack they need to judge.
+
+### Presentations
+This is the simplist approach to judging, every team presents their hack to all attendees and judges with a small, 5-10 minute, slot. Judges will see all hacks but this can become incredibly time consuming for large hackathons.
```
Comment (robzwolf) [inline docs/organise/before/judging.md]: I would definitely change the numbers here. Capping teams at 3 minutes is pretty common, and I'd say definitely no more than 5. If every team is presenting, you want to keep teams' presentations as short as possible.

File: docs/organise/before/judging.md [hunk 5]
```diff
@@ -1,16 +1,30 @@
...
+
+### Science Fair
+Here all the hacks set up in effect a stall to show off their hack, judges will visit a subset of all hacks and report back to select a top "**n**" that then present their hacks to the entire hackathon. Benefits from immense parallelisation but can create unruly and biased judging reports if the judges marking subset A are all marking higher than those marking subset B. [Gavel](https://github.com/anishathalye/gavel) can alliviate this by using comparative judging. In any case how to organise the judge's routes should be considered with simple assignments based off of devpost being available [here](https://github.com/nealrs/expo).
```
Comment (robzwolf) [inline docs/organise/before/judging.md]: `devpost` needs a capital `D`

File: docs/organise/before/stash_and_swag.md [hunk 6]
```diff
@@ -2,17 +2,19 @@
 
 Hackers love stickers!
 
-If you're with Major League Hacking, you will have a dedicated "I Demoed" Hexagon sticker along with MLH Swag stickers and Githib stickers as well.
+If you're with Major League Hacking, you will have a dedicated "I Demoed" Hexagon sticker along with MLH Swag stickers and Github stickers as well.
```
Comment (robzwolf) [inline docs/organise/before/stash_and_swag.md]: `Github` has a capital `H`

**Comment by AC-Aksan** [comment]
Thanks @robzwolf the joys of having done this with no spell check. All the notes seem solid and I'll bring them in when I'm back at a keyboard tomorrow.

**Comment by AC-Aksan** [comment]
Updated with those changes @robzwolf

**Pull request merged**